packages/web-config-server/src/apiV1/view.js

```diff
@@ -113,7 +109,12 @@ export default class extends DataAggregatingRouteHandler {
 
   // common view translation (for all possible views)
   addViewMetaData = inJson => {
-    const { viewJson, query, startDate, endDate } = this;
+    const {
+      viewJson,
+      query,
+      startDate = DEFAULT_MIN_DATE,
+      endDate = utcMoment().format('YYYYMMDD'),
```

Comment:
The front-end uses these values as defaults for start and end dates, so we may as well as specify them earlier on in the data flow. Ideally we would infer them from the results, but it's hard to do so for all data builders given the existing architecture. The new `reports` package will probably make this easier!